src/services/users.js

```diff
@@ -1,10 +1,12 @@
 const jwt = require("jsonwebtoken")
-const { genPasswordHash, verifyPassword } = require("../lib")
+const { genPasswordHash, verifyPassword, emailLib } = require("../lib")
```

Comment:
so @itzgeebee looks good but let's start our testing process.  this file keeps growing but we'll like to have a good concept of everything easily so first let's do this:

1. create a `users` folder
2. move this code into it and rename the file to `index.js`
3. then instead of adding the `inviteUser` here. move it to its own file `invite-user.js` and  import it here and export it like we do for the `src/lib/index.js`. just the invite-user though
4. create an `invite-user.test.js` and that's where we will begin the testing

kindly let me know

Reply:
Okay boss @jocrah

Reply:
am I going to write the test with this issue too @jocrah

Reply:
yea moving forward, the test is now going to be part of the issue, not separate. we're only putting automated checks in place to make sure everything works and if any subsequent updates break the code, we can know as well

Reply:
@itzgeebee

Reply:
actually, its fine for now. i think there's a bit more to do before to make the testing starts so i can approve this without the test.